x/distribution/legacy/v043/helpers.go

```diff
@@ -19,7 +19,8 @@ func MigratePrefixAddress(store sdk.KVStore, prefixBz []byte) {
 
 	for ; oldStoreIter.Valid(); oldStoreIter.Next() {
 		addr := oldStoreIter.Key()
-		newStoreKey := append(prefixBz, address.MustLengthPrefix(addr)...)
+		var newStoreKey []byte = prefixBz
+		newStoreKey = append(newStoreKey, address.MustLengthPrefix(addr)...)
```

Comment:
Nice! I don't think this was showing up as an error in the CI check because you have a new version of golangci-lint installed on your computer. I opened issue #9466 to prevent this in the future.

Reply:
I'm using :
osboxes@osboxes:~/Home/cosmos-sdk$ golangci-lint version
golangci-lint has version 1.40.1 built from 625445b1 on 2021-05-14T11:44:45Z